nifi-registry/nifi-registry-core/nifi-registry-resources/src/main/resources/conf/bootstrap-aws.conf

```diff
@@ -0,0 +1,28 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# AWS KMS Key ID is required to be configured for AWS KMS Sensitive Property Provider
+aws.kms.key.id=
+
+# First attempts to use credentials/configuration in bootstrap-aws.conf.
+# If credentials/configuration in bootstrap-aws.conf is not fully configured,
+# attempt to initialize credentials using default AWS credentials/configuration chain.
+
+# Optional AWS KMS Client Configuration/Credentials (all of them must be set in order to be used)
```

Comment:
Recommend rewording to clarify the implementation:
```suggestion
# NiFi uses the following properties when authentication to AWS when all values are provided.
# NiFi uses the default AWS credentials provider chain when one or more or the following properties are blank
# AWS SDK documentation describes the default credential retrieval order:
# https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html#credentials-chain
```

Reply:
Will make the changes